vtex/faststore
# SEO | Add noindex to non-static search pages

## What's the purpose of this pull request?
Improve SEO by reducing the number of search pages indexed by google, thus not breaking GoogleBot Budget

## How it works? 
Add `<meta name="robots" content="noindex" />` to non pre-rendered search pages. Pre rendered search pages are still indexed. 

Pages that are still indexed are:
1. All category tree since we are now pre-rendering all of them
2. Any landing page containing a search

Pages that are blocked from indexing:
1. category pages with filters
2. All full text search

## How to test it?
Explore the category tree. Any page in the category tree should be indexable. Adding any filter to the category should add the 
`<meta name="robots" content="noindex" />` to the html's head

[marinbrasil](https://github.com/vtex-sites/marinbrasil.store/pull/312)
[storecomponents](https://github.com/vtex-sites/storecomponents.store/pull/441)
[btglobal](https://github.com/vtex-sites/btglobal.store/pull/5)

## References
<em>Spread the knowledge: is there any content you used to create this PR that is worth sharing?</em>  

<em>Extra tip: adding references to related issues or mentioning people important to this PR may be good for the documentation and reviewing process</em>
